packages/web/server/api/edge-routers/assets-router.ts

```diff
@@ -51,6 +54,40 @@ export const assetsRouter = createTRPCRouter({
         userOsmoAddress,
       });
     }),
+  getAssetGasPriceStep: publicProcedure
+    .input(
+      z.object({
+        chainId: z.custom<MainnetChainIds>((val) => typeof val === "string"),
```

Comment:
Question: is using z.custom so you can cast `MainnetChainIds` type? Why not string array?

Reply:
We already have some validation in the procedure, and this avoids extra computation during runtime